src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -791,3 +791,33 @@ export const AutomatedAnalysisCard: React.FunctionComponent<AutomatedAnalysisCar
     </Card>
   );
 };
+
+export const AutomatedAnalysisCardDescriptor: DashboardCardDescriptor = {
+  title: 'Automated Analysis',
+  description: `
+Assess common application performance and configuration issues.
+    `,
+  descriptionFull: `
+Creates a recording and periodically evalutes various common problems in application configuration and performance.
+Results are displayed with scores from 0-100 with colour coding and in groups.
+This card should be unique on a dashboard.
+      `,
+  component: AutomatedAnalysisCard,
+  propControls: [
+    {
+      name: 'Compact Style',
+      key: 'isCompact',
+      description: 'Apply PatternFly compact Card styling to reduce padding',
+      kind: 'boolean',
+      defaultValue: true,
+    },
+    // TODO remove one or both of these, they are useful for development testing but not for a user
```

Comment:
I assume later, the actual AutomatedAnalysis config would be displayed here some sort of prop controls. Maybe having a boolean switch button that says "Use default config" and then when it gets switched to "off", then the AutomatedAnalysisConfig drawer may show up and somehow give config props back to the `AddCard` parent component for the Card.

Reply:
Makes sense. The configuration setup right now is much more basic than that and only allows configurations to be added that generate a simple form with simple controls, so the "default config" switch is already doable but embedding the config drawer is not yet possible. It could be done though, and in that particular instance I think it shouldn't be too hard since that component already uses Redux rather than needing to pass state back to the form or to the card component props.

Reply:
On this note, I'm thinking of adding an optional parameter to the `DashboardCardDescriptor` like `advancedConfig?: JSX.Element`. If this is present then the AddCard Wizard would either add a third config step, or maybe combine the basic config and advanced config into substeps of the config step, or something along these lines. The config element would be something like the AA card's drawer or form that can just be embedded here, and like before, it would be expected to persist state into redux/localstorage rather than being used to specify props.

Reply:
Substeps make sense. 

If we have three config steps, it would make me think that the two different config steps are for different functions. For example step 2 only configures the look of the card, and step 3 only configures the data that backs that card.

Reply:
On this note, we can have an expandable section like this?

Opened:

![Screenshot from 2022-12-12 20-37-12](https://user-images.githubusercontent.com/68053619/207204764-2ce9d6ee-eee0-48ff-bdbb-ff92e54c32fd.png)

Closed:

![Screenshot from 2022-12-12 20-36-58](https://user-images.githubusercontent.com/68053619/207204768-1a77ee18-a9af-4cb5-b81c-6230121e9605.png)

Reply:
I hesitate to add it as an expandable section since the substep functionality is right there already, so it seems like that would make things feel less cluttered and cramped into the wizard which is already boxed in inside a non-fullscreen card.

Right now it's implemented as a progressive wizard rather than substeps but it shouldn't be *too* hard to update that. The custom nav function that allows for the config steps to be optional just needs to do the same thing for substeps as well, I think.

FWIW I think most cards will only need to use the first "basic props" config step anyway, since the only options would be things like a chart or metric name/ID and then the sizing properties (ex. grid width). Most of the kinds of cards I envision just do and present one kind of information and just repeat that function for the current selected target, so they may not have any configuration at all. The AA card is a bit of a special case since it has to carry config for starting recordings too. Maybe we need something vaguely similar for the Grafana charts for uploading recordings to `jfr-datasource`.